README.md

```diff
@@ -948,5 +950,35 @@ See the [CONTRIBUTING.md] document.
 
 [CONTRIBUTING.md]: https://github.com/lynndylanhurley/devise_token_auth/blob/master/.github/CONTRIBUTING.md
 
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](https://github.com/lynndylanhurley/devise_token_auth/blob/master/.github/CONTRIBUTING.md)].
+<a href="graphs/contributors"><img src="https://opencollective.com/devise_token_auth/contributors.svg?width=890&button=false" /></a>
+
+
+## Backers
+
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/devise_token_auth#backer)]
+
+<a href="https://opencollective.com/devise_token_auth#backers" target="_blank"><img src="https://opencollective.com/devise_token_auth/backers.svg?width=890"></a>
+
+
+## Sponsors
+
+Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [[Become a sponsor](https://opencollective.com/devise_token_auth#sponsor)]
+
+<a href="https://opencollective.com/devise_token_auth/sponsor/0/website" target="_blank"><img src="https://opencollective.com/devise_token_auth/sponsor/0/avatar.svg"></a>
```

Comment:
Why all of this?

Reply:
The sponsor breakouts are so that each sponsor gets a logo with a link back to their website - it is treated different by default from backers....with the assumption that sponsors would be businesses. You can delete it if not applicable to you.

Reply:
I mean, why do we need 10 images?

Reply:
ok. Good luck. Also, you can chat with other collectives on the slack channel if you want to get an idea of what other folks are doing. Cheers.